status-EIPs.md

```diff
@@ -0,0 +1,211 @@
+# Status EIPs standards 
```

Comment:
Could we add a link to this in the README and from the main client spec as well?

Reply:
Added to README. In main client spec I am not sure where this link fits.

Reply:
Added commit with reference